x/leverage/keeper/liquidate.go

```diff
@@ -190,12 +195,13 @@ func ComputeLiquidation(
 // borrowed value can currently be repaid in a single liquidate transaction.
```

Comment:
```suggestion
// borrowed value can currently be repaid in a single liquidate transaction.
// We interpolate the maximum liquidation size based on the position of borrowed value between LiquidationThreshold and CollateralValue:
// B = borrowed value
// C = collateral value
// L = liquidation threshold (must be < C)
// CV = critical value - place between L and C, where 100% liquidation is allowed.
//    It is calculated as: L + (C-L)*completeLiquidationThreshold
// when B >= L liquidation is allowed
// when L <= B <= CV we interpolate the liquidation factor 
//  ------ | ---- | -- | ---- | ----
//         L      B    CV     C
```

Reply:
Does this work with godoc [formatting rules](https://go.dev/blog/godoc)? Specifically wondering if the lines starting with `B =`, `C =`, etc will be squashed together.

Reply:
you can apply changes and run godoc to see. 
Here is the full documentation: https://tip.golang.org/doc/comment